src/uu/mv/src/mv.rs

```diff
@@ -32,23 +32,56 @@ use fs_extra::dir::{
 };
 
 use crate::error::MvError;
-
-pub struct Behavior {
-    overwrite: OverwriteMode,
-    backup: BackupMode,
-    suffix: String,
-    update: UpdateMode,
-    target_dir: Option<OsString>,
-    no_target_dir: bool,
-    verbose: bool,
-    strip_slashes: bool,
-    progress_bar: bool,
+/// Options contains all the possible behaviors and flags for mv.
+///
+/// All options are public so that the options can be programmatically
+/// constructed by other crates, such as nushell. That means that this struct
+/// is part of our public API. It should therefore not be changed without good
+/// reason.
+/// The fields are documented with the arguments that determine their value.
+#[derive(Debug, Clone, Eq, PartialEq)]
+pub struct Options {
+    /// specifies overwrite behavior
+    /// '-n' '--no-clobber'
+    /// '-i' '--interactive'
+    /// '-f' '--force'
+    pub overwrite: OverwriteMode,
+
+    /// `--backup[=CONTROL]`, `-b`
+    pub backup: BackupMode,
```

Comment:
I think `cp` re-exports the `BackupMode` and I think we could do the same here. Then nushell doesn't have to use `uucore` directly.